node/src/components/event_stream_server/sse_server.rs

```diff
@@ -62,6 +62,11 @@ pub enum SseData {
         #[data_size(skip)]
         execution_result: Box<ExecutionResult>,
     },
+    Equivocation {
```

Comment:
@afck Should it be called `Equivocation` or `Evidence`? Isn't `Equivocation` Highway-specific? Or specific to some family of consensus protocols.

Reply:
I'd also prefer `Evidence`… or `Fault`?

Reply:
Renamed to Evidence